core/src/main/java/com/netflix/conductor/metrics/Monitors.java

```diff
@@ -169,6 +169,10 @@ public static void recordGauge(String name, long count) {
         gauge(classQualifier, name, count);
     }
```

Comment:
Instead of ENV_DEFAULT_LOG_LEVEL, to enable/disable metrics logging we can use existing property - conductor.metrics-logger.enabled
https://github.com/Netflix/conductor/blob/2ccf09e1a31bf4e5ebf80982199250cf82f50556/CHANGELOG.md#component-configuration

Reply:
lets see what others also saying. if debug is enabled, always metrics are added in the log. but if we add a specific parameter then it has to be added as a bender configuration so that it can be edited on editing sts file. not a spring configuration in galaxy.